keps/sig-api-machinery/2876-crd-validation-expression-language/README.md

```diff
@@ -231,22 +231,13 @@ element of the list or map.
   
 - For immutability use case, validator will have access to the existing version of the object. 
 
-<<[UNRESOLVED @jpbetz @deads2k @lavalamp]>>
-When performing an update, should the live state of a field also be readable by expressions? This could be used,
-for example, to enforce immutability.
-<<[/UNRESOLVED]>>
-
-<<[UNRESOLVED @cici37]>>
-Should the message also be an expression to allow for some basic variable
-substitution?
-<<[/UNRESOLVED]>>
-
-<<[UNRESOLVED @cici37]>>
-Should a 'type' field also be required? If it is not 'cel', the validator
-could be skipped to future proof against addition of other ways to validate in
+- Have a `type` field specified with `cel` within `x-kubernetes-validator`.
```

Comment:
To minimize verbosity of using the new feature, how about we move this to future work?  If we ever add a second language we can add the type field and default it to CEL for backward compatibility.